packages/node/src/utils/substrate.ts

```diff
@@ -90,6 +95,22 @@ export function getTimestamp({
   return undefined;
 }
 
+export async function getTimestampFromBlockHash(
```

Comment:
This seems to be very unreliable. It doesn't work for AssetHub. I think we should use the other way of getting block timestamps.